lib/class_board.rb

```diff
@@ -0,0 +1,10 @@
+# this the display board methods
+def display_board_game(arr)
+  system 'clear'
+  puts "  |  #{arr[6]}  |  #{arr[7]}  |  #{arr[8]}  |"
+  puts '  |     |     |     |'
+  puts "  |  #{arr[3]}  |  #{arr[4]}  |  #{arr[5]}  |"
+  puts '  |     |     |     |'
+  puts "  |  #{arr[0]}  |  #{arr[1]}  |  #{arr[2]}  |"
+  puts '  |     |     |     |'
+end
```

Comment:
- [ ] **[OPTIONAL]** This step was intended to be mandatory. There should be **no** `puts`/`print` and `get.chomp` outside main.rb file. Please fix it.
![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png)